main/test/ca/uwaterloo/flix/library/Core/System/TestEnvironment.flix

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Ramiro Calle
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use Core/System/Environment.{getEnv, getVar, getProp, getOsName, getOsArch, getOsVersion, getFileSeparator, getPathSeparator, getLineSeparator, getCurrentWorkingDirectory, getTemporaryDirectory, getUserName, getUserHomeDirectory};
+
+@test
+def testGetEnv01(): Bool & Impure =
```

Comment:
Testing this is indeed tricky. Do you have any ideas about how to make these tests more robust?

Reply:
I think these tests are fine, I would just ask:

- Move them into the `TestEnvironment` namespace.
- Instead of using all the functions, I would just call them directly `Environment.getOsName`.
- No reason do to do `foo == true`. Simply return the Boolean or its negation.

Reply:
Hi, 
@mlutze, To make the tests more robust I've created a temporary file containing all environment variables at the `scalatest` level and check if we can find the same values at the flix side.

@magnus-madsen I don't know (yet) how to call directly the functions `Environment.getOsName` since they are in the namespace `Core/System/Environment`.

Reply:
@magnus-madsen ignore my last comment. I'll move `Environment` to the root.